bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -119,6 +119,12 @@ public void startCheckpoint(Checkpoint checkpoint) {
         });
     }
 
+    void start() {
```

Comment:
since 'flushInterval' variable is already there in this class, I used it. 

Anyhow, I'll make the changes as you suggested. Also I'm planning to use "scheduleWithFixedDelay" instead of scheduleAtFixedRate, so that way it is more certain that interval between two executions is 'flushInterval'.